Source/Particle.cpp

```diff
@@ -1,62 +1,51 @@
-#include <iomanip>
-#include <vector>
-#include <algorithm>
-#include <string>
 
-#include "PeleC.H"
-
-using namespace amrex;
 
-#ifdef PELEC_SPRAY
+#ifdef PELEC_USE_SPRAY
+#include "PeleC.H"
+#include "SprayParticles.H"
 
 namespace {
 bool virtual_particles_set = false;
 
 // Containers for the real "active" Particles
-SprayParticleContainer* SprayPC = 0;
+SprayParticleContainer* SprayPC = nullptr;
```

Comment:
can we make these three pointers to be `unique_ptr` ?

Reply:
I will have to move the pointers out of the namespace in `Particle.cpp` to make the pointers member data for PeleC. I guess this would actually be less messy though. I will go ahead and try it out.